examples/nucleo_h723zg/vl53/vl53l5cx_i2c/project.xml

```diff
@@ -0,0 +1,15 @@
+<library>
+  <extends>modm:nucleo-h723zg</extends>
+  <options>
+    <option name="modm:build:build.path">../../../../build/nucleo_h723zg/vl53l5cx_i2c</option>
+    <option name="modm:vl53:vl53l5cx:examples">yes</option>
+    <option name="modm:platform:cortex-m:main_stack_size">64Ki</option>
```

Comment:
WAT

Reply:
The I2C version needs the 64kB (or at least it stops working if I set 32kB stack, not really sure how to debug stack usage in modm.
The SPI version needs only 16kB (depending on the size of the buffer for the SPI read/write).

The firmware needs a bunch of large buffers for the transfers and also has pretty large firmware blobs (`cost uint8_t[LARGE]`), not sure where this breaks with smaller stack.